### ISSUE TITLE
Languages: independent quality fields

### ISSUE THREAD

**Issue by AntoineMarnat** [c2corg/c2c_ui#1554]
**Is your feature request related to a problem? Please describe.**
The quality field is associated to a document, and the same for all language versions. This avoids featuring which is the main language (ie highest quality), that would have many usefull application.

**Describe the solution you'd like**
The quality field should be pushed in the locale, hence depend on the language.

**Applications**
- Add the quality logo next to the languages, when there is different languages. See e.g. https://www.camptocamp.org/routes/54424/.
- When the document does not exist in the language of the interface,
   - suggest to create a version in the language of the interface in the language area
   - choose a language to display with highest quality.

**Comment by AntoineMarnat** [comment]
I am affraid, this requires API change.

**Comment by asaunier** [comment]
@AntoineMarnat Right, there is an old API issue about this at https://github.com/c2corg/v6_api/issues/701
